src/main/java/com/example/QuizGame/service/ScoreService.java

```diff
@@ -0,0 +1,41 @@
+package com.example.QuizGame.service;
+
+import com.example.QuizGame.model.Score;
+import com.example.QuizGame.repository.ScoreRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.sql.Timestamp;
+import java.util.Date;
+
+@Service
+public class ScoreService {
+
+    private final ScoreRepository scoreRepository;
+
+    @Autowired
+    public ScoreService(ScoreRepository scoreRepository) {
+        this.scoreRepository = scoreRepository;
+    }
+
+    // Logic of updating the high score for a user
```

Comment:
Why do not use javadoc?

Reply:
Javadoc will be implemented after this pr